### DISCUSSION TITLE
Midnight Roll Up

### DISCUSSION THREAD

**Discussion by jdigangi** [jertel/vuegraf#168] [General]
Does anyone know how to disable the daily roll up that happens at midnight ?

**Comment by jdigangi** [comment]
},
  "updateIntervalSecs": 60,
  "detailedDataEnabled": true,
  "detailedDataSecondsEnabled": false,
  "detailedDataHoursEnabled": false,
  "timezone": "America/New_York",
  "accounts": [
    {

Every day at midnight I get this roll up, I am trying to disable it with no luck any help is much appreciated


<img width="500" alt="image" src="https://github.com/user-attachments/assets/6db580b4-d5af-48bf-bc4f-31cad4716188">

**Comment by jertel** [comment]
There is no way currently to disable it.

It only adds a handful of datapoints to the DB each day. 

I see a graph, and I see a spike a midnight, but I don't know what that is showing. 

Can you explain specifically why you need it disabled? E.g, are you thinking it's skewing the rest of your data?

**Comment by jdigangi** [comment]
It skews my graphs with a large value at midnight daily and makes the long term view kinda unusable.  I use this to visualize my circuits that I am monitoring 

7 day view

￼


6hr view 
￼

I was messing with the 3 switches 

"updateIntervalSecs": 60,
  "detailedDataEnabled": true,
  "detailedDataSecondsEnabled": true,
  "detailedDataHoursEnabled": true,

I have to check further but it looks like I lost my main breaker monitor also….that was in green above


I really appreciate you responding your software is pretty awesome!

Thanks

**Reply by jertel** [reply]
Thanks for the kind words.

Your graphs aren't showing up; could be a github issue. Meanwhile, are you using the pre-made dashboard that's included with this project? Or are you using your own graph. I ask because I use the dashboard bundled with this project and I don't have any spikes like that. 

Ex: 
![image](https://github.com/user-attachments/assets/a6fba98e-7bb3-4e38-9e63-0f04f95a81b1)

This is a 7-day view across all circuits, and the large peaks are the EV charger. There aren't any narrow spikes at the midnight slot.

**Comment by jdigangi** [comment]
I am putting the data into influx and using grafana to display it.  I will post some screen shots

**Comment by jdigangi** [comment]
<img width="1421" alt="image" src="https://github.com/user-attachments/assets/8f75c108-0940-4d77-8cc2-a983e5b147c6">
 Data is going into influx 1.8 and using grafana to display

**Comment by jertel** [comment]
It looks like you have a similar problem with hourly fetches as well. I suspect your Grafana queries aren't properly accounting for the **daily** and **hourly** data, and perhaps even the **second** data. I haven't used Grafana in several years so I don't have a working query for it. You will need to look at the flux queries included with this project and figure out how to replicate the behavior in Grafana.

**Comment by jdigangi** [comment]
Spot on I re-wrote my query and things look better, lets give it 24hrs Thanks again!
